test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -430,3 +432,13 @@ def mark_as_read(self, chatName: str):
             test.fail("Chat is not loaded")
         
         click_obj_by_name(CommunityScreenComponents.MARK_AS_READ_BUTTON.value)
+        
+    def click_sidebar_option(self, community_sidebar_option:str):
+        #TODO Make compatible with other sidebar options
+        if community_sidebar_option=="Manage Community":
```

Comment:
```suggestion
        if community_sidebar_option == "Manage Community":
```

Reply:
Maybe we can create an enum for the sidebar options instead of direclty compare to a string